ivy_tests/test_ivy/test_frontends/test_numpy/test_matrix/test_methods.py

```diff
@@ -193,7 +237,7 @@ def test_numpy_matrix_argmax(
     ),
     to_str=st.booleans(),
```

Comment:
you may add an additional data generation here for better testing the `dtype` argument of the method, eg, `helpers.get_dtypes("numeric", full=False)`, this will give you a single dtype for passing into the method